### PR TITLE
feat(CI/MacOS): use acore.sh to install deps

### PR DESCRIPTION
**Pull request by locus313**
## CHANGES PROPOSED:
-  use acore.sh to install deps for macos ci build

## ISSUES ADDRESSED:
- Closes none

## KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: apps/bash_shared/defines.sh [hunk 1]
```diff
@@ -1,5 +1,14 @@
 unamestr=$(uname)
 if [[ "$unamestr" == 'Darwin' ]]; then
+   if ! command -v brew &>/dev/null ; then
+       ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+   fi
+   if ! [ "${BASH_VERSINFO}" == 5 ]; then
```
Comment (FrancescoBorzi) [inline apps/bash_shared/defines.sh:6]: could be lower than ;)

Comment (locus313) [reply apps/bash_shared/defines.sh]: looks like it needs to be atleast version 4. so I changed it to check if its 4 or greater

File: .github/workflows/core_build.yml [hunk 2]
```diff
@@ -85,8 +85,10 @@ jobs:
           restore-keys: |
             ccache:${{ matrix.os }}:${{ github.ref }}
             ccache:${{ matrix.os }}
+      - name: Install latest bash
+        run: brew install bash
       - name: Configure OS
-        run: source ./apps/ci/mac/ci-install.sh
+        run: source ./acore.sh install-deps
```
Comment (FrancescoBorzi) [inline .github/workflows/core_build.yml:91]: we can now remove `/apps/ci/mac/ci-install.sh`, can't we?

Comment (locus313) [reply .github/workflows/core_build.yml:91]: yep, sure can. added the removal of /apps/ci/mac/ci-install.sh to the pr.

**Review by FrancescoBorzi — APPROVED** [review]

**Review by Si1ker — APPROVED** [review]

**Pull request merged**